src/models/nearest_neighbors.py

```diff
@@ -16,7 +16,7 @@ def forward(self, xs, ys):
 
         for i in range(ys.shape[1]):
             if i == 0:
-                preds.append(torch.zeros_like(ys[:, 0]))  # predict zero for first point
+                preds.append(torch.zeros(xs.shape[:1] + torch.Size([1, self.y_dim]), device=xs.device))  # predict zero for first point
```

Comment:
is the 1 here so that it can be broadcasted, or am I misunderstanding how this line works?

Reply:
This is for `torch.cat` later down. We need the dimension present to concatenate with predictions later on. This line predicts a constant 0 for the first point in every sequence.